network-api/networkapi/wagtailpages/customblocks.py

```diff
@@ -488,3 +488,21 @@ class Meta:
         template = 'wagtailpages/blocks/profile_blocks.html'
         icon = 'group'
         value_class = LatestProfileQueryValue
+
+
+class ProfileDirectory(LatestProfileList):
+    filter_values = blocks.CharBlock(
+        required=True,
+        default='',
+        help_text='Example: 2013,2014,2015,2016,2017,2018'
+    )
+
+    def get_context(self, value, parent_context=None):
```

Comment:
This is a bit of a funky thing: the component lets people type strings, but we want to work with an array of strings in the template, so we override the definition of `get_context`, which is the dictionary that django uses as "variables available inside templates", and then we add in a context variable called `filters` which is the original string, split up into an array instead.